@xen-orchestra/fs/src/abstract.js

```diff
@@ -84,6 +86,10 @@ export default class RemoteHandlerAbstract {
     return this.#rawEncryptor
   }
 
+  get immutability(){
+    return this.#immutability
+  }
+
```

Comment:
IMHO, this should not be in `fs`, move into `async RemoteAdapter.getImmutability()` and it can have a cache.